components/raftstore/src/store/async_io/write.rs

```diff
@@ -535,7 +535,13 @@ where
                         self.store_id, self.tag, e
                     );
                 });
-            self.perf_context.report_metrics(&[]); // TODO: pass in request trackers
+            let trackers: Vec<_> = self
+                .batch
+                .tasks
+                .iter()
+                .flat_map(|task| task.trackers.iter().flat_map(|t| t.as_tracker_token()))
+                .collect();
+            self.perf_context.report_metrics(&trackers);
```

Comment:
Well, this is some mess now. Raft engine does not have perf contexts, so I just use RocksDB's perf context now. If the user still uses RocksDB as the raft engine, this will still record related information.

I'm trying to add perf context to raft-engine: https://github.com/tikv/raft-engine/pull/227